### ISSUE TITLE
Spacecore Chaos

### ISSUE THREAD

**Issue by KawaiFoxHappyClaws76** [ZaneYork/SMAPI-Android-Installer#423]
İdk why but Spacecore is updated with writed desc. about android smapi but Always Crashes .... İn short Big Ooof

**Comment by KawaiFoxHappyClaws76** [comment]
And CJB mods as well (İtemSpawner Cheat menu and ShowİtemPrice mod)

**Comment by ArthurWarden** [comment]
Use 1.27.0 version of cjb cheats menu, it should work. Can't say about the other mods since i stopped testing them, but always try the older versions if the latest isn’t working.

**Comment by KawaiFoxHappyClaws76** [comment]
> Use 1.27.0 version of cjb cheats menu, it should work. Can't say about the other mods since i stopped testing them, but always try the older versions if the latest isn’t working.

Yeah i saw Cheat menu problem on nexus and yes work but item dpawner not works but pathoschild has a fix i guess ...

**Comment by KawaiFoxHappyClaws76** [comment]
and showitem price work as well i guess

**Comment by Defros89** [comment]
Is there maybe a working version of chests anywhere?

**Comment by 3Negativ3** [comment]
> Is there maybe a working version of chests anywhere?

Latest version works just fine

**Comment by Defros89** [comment]
> > Is there maybe a working version of chests anywhere?
> 
> Latest version works just fine

![Screenshot_2023-07-01-07-15-37-199_com zane stardewvalley](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/137830649/b2ed0ea0-699e-42d4-a6e2-02b3da977337)
All I see is this. No drop-down menu to change chests. Can I change the scaling of the mod? Maybe it's just outside of the screen?
